### DISCUSSION TITLE
New Label: Low Priority

### DISCUSSION THREAD

**Discussion by Tyler-Keith-Thompson** [wwt/SwiftCurrent#44] [Ideas] [answered]
Some open source projects have a "Low Priority" label they'll use. I feel this is an effective label because then when I look at issues I can see that they've been acknowledged, the team may even agree with them, but the team has more important work to do. 

This indicates to me that as a contributor I have several options at my disposal, if I feel it shouldn't be low priority and it's important I can add my voice to it. If I want more immediate results I can submit a PR for it.

I propose we add this label to our list and make it a standard thing we put on issues. It will also explain why some issues are immediately closed and some hang around, it's not that we're inactive, it's that we've got more important work to do.

**Comment by Richard-Gist** [comment]
This makes sense to us.   We'll get one added and start applying it to things.

**Comment by Richard-Gist** [comment]
I marked this discussion with the label created.

**Comment by morganzellers** [comment, marked as answer]
We've added this label, so I am going to lock this discussion

**Reply by Tyler-Keith-Thompson** [reply]
What does locking do?!? Cause I can totally reply

**Reply by morganzellers** [reply]
Looks like members with write access to the repo can still comment after its locked, so...doesn't do much lol
